app/views/admin/_admin_nav.html.erb

```diff
@@ -6,6 +6,11 @@
   <li>
     <%= span_if_current ts('Post AO3 News', key: 'header'), new_admin_post_path %>
   </li>
+  <% if current_page?(action: 'edit') %>
```

Comment:
Because this only specifies the action and not the action and the model, the Delete Post button will also show up on edit pages for Wrangling Guidelines, FAQs, and Known Issues.